shell/platform/windows/flutter_windows_engine.cc

```diff
@@ -189,6 +194,18 @@ void FlutterWindowsEngine::SetSwitches(
   project_->SetSwitches(switches);
 }
 
+double ApproxFrameRate() {
```

Comment:
I suggest using time per frame instead of frame rate and use `std::chrono::duration` instead.